api/v2/openapi.yaml

```diff
@@ -0,0 +1,345 @@
+---
+
+swagger: '2.0'
+
+info:
+  version: 0.0.1
+  title: Alertmanager API
+  description: API of the Prometheus Alertmanager (https://github.com/prometheus/alertmanager)
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+
+paths:
+  /status:
+    get:
+      tags:
+        - general
+      operationId: getStatus
+      description: Get current status of an Alertmanager instance and its cluster
+      responses:
+        '200':
+          description: Status response
+          schema:
+            $ref: '#/definitions/alertmanagerStatus'
+  /receivers:
+    get:
+      tags:
+        - receiver
+      operationId: getReceivers
+      description: Get list of all receivers (name of notification integrations)
+      responses:
+        '200':
+          description: Receivers response
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/receiver'
+  /silences:
+    get:
+      tags:
+        - silence
+      operationId: getSilences
+      description: Get a list of silences
+      responses:
+        '200':
+          description: List of silences response
+          schema:
+            $ref: '#/definitions/silences'
+        '500':
```

Comment:
Why does this explicitly have 500 called out, and others don't?

Reply:
Whenever a handler e.g. `getSilencesHandler` needs to return a specific HTTP error code, it needs to be mentioned in the respective `path` definition in the OpenAPI specification.

Others don't have 500 explicitly called out, because their handlers do not return a 500 error code.